config/urls.py

```
@@ -5,11 +5,14 @@
 from rest_framework.permissions import AllowAny
 from drf_yasg.views import get_schema_view
 from drf_yasg import openapi
+from django.conf import settings
```

Ruff (0.12.2)

Redefinition of unused `settings` from line 3

Remove definition: `settings`

(F811)